File: app/src/main/java/hyunwook/co/kr/paging_livedata/entity/Person.java
package hyunwook.co.kr.paging_livedata.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Person {
    @PrimaryKey (autoGenerate = true)
    private int uid;

    @ColumnInfo(name = "first_name")
    private String firstName;

    @ColumnInfo(name = "last_name")
    private String lastName;


}

File: app/src/main/java/hyunwook/co/kr/paging_livedata/entity/Person.kt
package hyunwook.co.kr.paging_livedata.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Person(@PrimaryKey (autoGenerate = true) val uid: Int, @ColumnInfo (name = "first_name") val firstName: String, @ColumnInfo (name = "last_name") val lastName : String)